### DISCUSSION TITLE
git clone error when using lfs

### DISCUSSION THREAD

**Discussion by fugedizsolt** [community/community#47883] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

I use windows 10.
I have removed completely my 2.35 Git (deleted my gitconfig, everthing), then I have installed Git-2.39.2-64-bit.exe with LFS.
I started Git Bash, "git lfs install", -> initialized ok.

When I clone a git repository using no LFS, there is no problem.
I have a couple of repos using LFS, I cannot clone them all, and any repository using LFS.

Please help.
** additional information: "R:" drive is a ImDisk Ramdisk drive.
https://sourceforge.net/projects/imdisk-toolkit/
This git clone command worked with my previous 2.35 Git installation.

$ git clone -v --progress https://github.com/fugedizsolt/vrinput1.git
Cloning into 'vrinput1'...
warning: ----------------- SECURITY WARNING ----------------
warning: | TLS certificate verification has been disabled! |
warning: ---------------------------------------------------
warning: HTTPS connections may not be secure. See https://aka.ms/gcm/tlsverify for more information.
warning: ----------------- SECURITY WARNING ----------------
warning: | TLS certificate verification has been disabled! |
warning: ---------------------------------------------------
warning: HTTPS connections may not be secure. See https://aka.ms/gcm/tlsverify for more information.
POST git-upload-pack (185 bytes)
POST git-upload-pack (212 bytes)
remote: Enumerating objects: 1778, done.
remote: Counting objects: 100% (1778/1778), done.
remote: Compressing objects: 100% (949/949), done.
remote: Total 1778 (delta 878), reused 1723 (delta 825), pack-reused 0Receiving objects:  98% (1743/1778)
Receiving objects: 100% (1778/1778), 5.10 MiB | 11.02 MiB/s, done.
Resolving deltas: 100% (878/878), done.
Error: error converting "R:/GitHub/vrinput1/.git" to absolute: Incorrect function.
fatal: protocol error: bad line length character: Not
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @fugedizsolt ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
